src/tools/clippy/tests/ui/almost_complete_range.stderr

```diff
@@ -2,226 +2,303 @@ error: almost complete ascii range
+help: use an inclusive range
+   |
+LL |         b'0'..=b'9' if true => 3,
+   |             ~~~
 
 error: almost complete ascii range
   --> tests/ui/almost_complete_range.rs:49:9
    |
 LL |         'a'..'z' if true => 1,
-   |         ^^^--^^^
-   |            |
-   |            help: use an inclusive range: `..=`
+   |         ^^^^^^^^
+   |
+help: use an inclusive range
+   |
+LL |         'a'..='z' if true => 1,
+   |            ~~~
 
 error: almost complete ascii range
   --> tests/ui/almost_complete_range.rs:50:9
    |
 LL |         'A'..'Z' if true => 2,
-   |         ^^^--^^^
-   |            |
-   |            help: use an inclusive range: `..=`
+   |         ^^^^^^^^
+   |
+help: use an inclusive range
+   |
+LL |         'A'..='Z' if true => 2,
+   |            ~~~
 
 error: almost complete ascii range
   --> tests/ui/almost_complete_range.rs:51:9
    |
 LL |         '0'..'9' if true => 3,
-   |         ^^^--^^^
-   |            |
-   |            help: use an inclusive range: `..=`
+   |         ^^^^^^^^
+   |
+help: use an inclusive range
+   |
+LL |         '0'..='9' if true => 3,
+   |            ~~~
 
 error: almost complete ascii range
   --> tests/ui/almost_complete_range.rs:64:17
    |
 LL |         let _ = 'a'..'z';
-   |                 ^^^--^^^
-   |                    |
-   |                    help: use an inclusive range: `..=`
+   |                 ^^^^^^^^
    |
    = note: this error originates in the macro `__inline_mac_fn_main` (in Nightly builds, run with -Z macro-backtrace for more info)
+help: use an inclusive range
+   |
+LL |         let _ = 'a'..='z';
+   |                    ~~~
 
 error: almost complete ascii range
   --> tests/ui/almost_complete_range.rs:65:17
    |
 LL |         let _ = 'A'..'Z';
-   |                 ^^^--^^^
-   |                    |
-   |                    help: use an inclusive range: `..=`
+   |                 ^^^^^^^^
    |
    = note: this error originates in the macro `__inline_mac_fn_main` (in Nightly builds, run with -Z macro-backtrace for more info)
+help: use an inclusive range
+   |
+LL |         let _ = 'A'..='Z';
+   |                    ~~~
 
 error: almost complete ascii range
   --> tests/ui/almost_complete_range.rs:66:17
    |
 LL |         let _ = '0'..'9';
-   |                 ^^^--^^^
-   |                    |
-   |                    help: use an inclusive range: `..=`
+   |                 ^^^^^^^^
    |
    = note: this error originates in the macro `__inline_mac_fn_main` (in Nightly builds, run with -Z macro-backtrace for more info)
+help: use an inclusive range
+   |
+LL |         let _ = '0'..='9';
+   |                    ~~~
 
 error: almost complete ascii range
   --> tests/ui/almost_complete_range.rs:73:9
    |
 LL |         'a'..'z' => 1,
-   |         ^^^--^^^
-   |            |
-   |            help: use an inclusive range: `...`
+   |         ^^^^^^^^
+   |
+help: use an inclusive range
+   |
+LL |         'a'...'z' => 1,
+   |            ~~~
```

Comment:
This clippy suggestions are just plain wrong, right?

Reply:
It's the old pattern syntax for when MSRV is set to the ancient times

https://github.com/rust-lang/rust/blob/2b90614e94cfb400820cfc10fe63b0db74f9e67a/src/tools/clippy/tests/ui/almost_complete_range.rs#L70-L78